src/methods/LocalCompare.js

```diff
@@ -38,23 +47,25 @@ export default class LocalCompare extends BaseCompare {
 
       const diffPath = this.getDiffFile(context);
 
+
       if (misMatchPercentage > misMatchTolerance) {
         log(`Image is different! ${misMatchPercentage}%`);
         const png = compareData.getDiffImage().pack();
         await this.writeDiff(png, diffPath);
+        options.diff = diffPath;
 
-        return this.createResultReport(misMatchPercentage, false, isSameDimensions);
+        return this.createResultReport(misMatchPercentage, false, isSameDimensions, options);
```

Comment:
nit: the var name options is misleading here as these are report results not config options.